ogr/services/github/secrets.py

```diff
@@ -0,0 +1,36 @@
+# MIT License
+#
+# Copyright (c) 2018-2021 Red Hat, Inc.
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import logging
+from github.ApplicationOAuth import ApplicationOAuth as _GithubApplicationOAuth
+
+from ogr.services.base import BaseSecrets
+
+logger = logging.getLogger(__name__)
+
+
+class GithubSecrets(BaseSecrets):
+    _raw_auth: _GithubApplicationOAuth
+
+    @property
+    def secrets(self) -> str:
+        return self._raw_auth.client_secret
```

Comment:
Ben, I just clarified in the linked issue that we need [GitHub Actions Secrets](https://docs.github.com/en/rest/reference/actions#secrets).